<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QTextBrowser" name="textEditRecvive">
      <property name="styleSheet">
       <string notr="true">background-color: balck;
color: white;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="lineEditSend">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="buttonSend">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1038</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="menuLog">
     <property name="title">
      <string>日志</string>
     </property>
     <addaction name="actionStart"/>
     <addaction name="actionStop"/>
     <addaction name="actionPause"/>
     <addaction name="actionContinue"/>
     <addaction name="separator"/>
     <addaction name="actionOpenFile"/>
     <addaction name="actionOpenLog"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionRe_Connect"/>
    <addaction name="separator"/>
    <addaction name="menuLog"/>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionOtherSave"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
    <addaction name="separator"/>
    <addaction name="actionClear"/>
    <addaction name="actionClearCache"/>
    <addaction name="actionClear_All"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>查看</string>
    </property>
    <widget class="QMenu" name="menuFilter">
     <property name="title">
      <string>筛选</string>
     </property>
     <addaction name="actionBy_level"/>
     <addaction name="actionBy"/>
     <addaction name="actionBy_function"/>
     <addaction name="actionBy_file"/>
     <addaction name="actionDefine"/>
    </widget>
    <addaction name="actionFind"/>
    <addaction name="actionFind_All"/>
    <addaction name="separator"/>
    <addaction name="menuFilter"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actionOptions"/>
    <addaction name="separator"/>
    <addaction name="actionFast_cmd"/>
    <addaction name="actionClolor"/>
    <addaction name="separator"/>
    <addaction name="actionIPC_parse"/>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>选项卡</string>
    </property>
    <widget class="QMenu" name="menuShow">
     <property name="title">
      <string>显示</string>
     </property>
     <addaction name="actionTime"/>
     <addaction name="actionTag"/>
     <addaction name="actionFile"/>
    </widget>
    <addaction name="menuShow"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>窗口</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuOption"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>断开</string>
   </property>
  </action>
  <action name="actionRe_Connect">
   <property name="text">
    <string>重连</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>导入</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>导出</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionOtherSave">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>开始</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>停止</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>暂停</string>
   </property>
  </action>
  <action name="actionContinue">
   <property name="text">
    <string>继续</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string>打开文件夹</string>
   </property>
  </action>
  <action name="actionOpenLog">
   <property name="text">
    <string>打开日志</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>清屏</string>
   </property>
  </action>
  <action name="actionClearCache">
   <property name="text">
    <string>清缓存</string>
   </property>
  </action>
  <action name="actionClear_All">
   <property name="text">
    <string>清屏和缓存</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>查找</string>
   </property>
  </action>
  <action name="actionFind_All">
   <property name="text">
    <string>查找所有</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>选项</string>
   </property>
  </action>
  <action name="actionFast_cmd">
   <property name="text">
    <string>快速指令</string>
   </property>
  </action>
  <action name="actionClolor">
   <property name="text">
    <string>颜色</string>
   </property>
  </action>
  <action name="actionIPC_parse">
   <property name="text">
    <string>IPC解析</string>
   </property>
  </action>
  <action name="actionBy_level">
   <property name="text">
    <string>按等级</string>
   </property>
  </action>
  <action name="actionBy">
   <property name="text">
    <string>按标签</string>
   </property>
  </action>
  <action name="actionBy_function">
   <property name="text">
    <string>按函数</string>
   </property>
  </action>
  <action name="actionBy_file">
   <property name="text">
    <string>按文件</string>
   </property>
  </action>
  <action name="actionDefine">
   <property name="text">
    <string>自定义</string>
   </property>
  </action>
  <action name="actionTime">
   <property name="text">
    <string>时间</string>
   </property>
  </action>
  <action name="actionTag">
   <property name="text">
    <string>标签</string>
   </property>
  </action>
  <action name="actionFile">
   <property name="text">
    <string>文件</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
